lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -740,6 +797,88 @@ ParseResult FIRParser::parseOptionalRUW(RUWAttr &result) {
   return success();
 }
 
+ParseResult FIRParser::importAnnotations(const SMLoc &loc,
+                                         StringRef annotationsStr,
+                                         MLIRContext *context,
```

Comment:
No need to pass in a context, you can call `getContext()` in FIRParser.

Reply:
Nice catch. Updated in a `!fixup` commit.